ts-web-component/src/index.ts

```diff
@@ -1,6 +1,63 @@
+import {Replicache} from 'replicache';
+import {mutators} from '../../shared/mutators';
 import {MyTodo} from './component/my-todo';
 import {TodoItem} from './component/todo-item';
+import {createSpace, spaceExists} from './space';
 import './style.css';
 
 window.customElements.define('my-todo', MyTodo);
 window.customElements.define('todo-item', TodoItem);
+
+async function init() {
```

Comment:
replicache init is non DRY at the moment. I need to figure out the correct directory structure to handle this well. The current shared folder is for server-client shared files.